File: Split Strings/Split Strings.py
S="b1397cg1xjzm90j0j!!!@3"
S1=S2=S3=""
for i in S:
    if i.isdigit():
        S2+=i
    elif i.isalpha():
        S1+=i
    else:
        S3+=i
print (S1,S2,S3)

File: Split Strings/Split Strings.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;


int main(){
    string S = "nsd9ch189dh1nwon189ndc!!@#";
    vector<string> ans;
        string temp1,temp2,temp3;
        for(int i=0;i<S.length();i++){
            if(isalpha(S[i])){
                temp1.push_back(S[i]);
            }
            else if(isdigit(S[i])){
                temp2.push_back(S[i]);
            }
            else{
                temp3.push_back(S[i]);
            }
        }
        ans.push_back(temp1);
        ans.push_back(temp2);
        ans.push_back(temp3);
        cout<<ans;
}